### DISCUSSION TITLE
Episode 189: C++26 Reflection, std::execution & CppNorth

### DISCUSSION THREAD

**Discussion by codereport** [codereport/adsp2#88] [Episode]
Episode Link: https://adspthepodcast.com/2024/07/05/Episode-189.html

Feel free to write comments or ask questions 🙂

**Comment by pmcgee69** [comment]
" ... Ooooooooh Bryce ... " 😆

**Reply by pmcgee69** [reply]
and regarding YouTube ..
a. 100% great value from YT recommendations 
b.  I guess transcripts maybe came about because they were scanning through for copyrighted audio anyway(?)
c.  YT might do all of its video processing on the standard res versions, rather than the 4K 60fps.

**Comment by CharString** [comment]
Python decorators are just syntactic sugar:

```python

@foo
def bar():
    pass

# is the same as
def bar():
    pass

bar = foo(bar)
